indico/modules/events/registration/templates/pdf/participants_book.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+</head>
+
+<body>
+    <section class="title-page">
+        <div>{{ event.title }}</div>
+        <h1>{%- trans %}Registrants Book{% endtrans -%}</h1>
+    </section>
+
+    <section class="toc-page">
+        <h2>{%- trans %}Table of Contents{% endtrans -%}</h2>
+        {% for row in rows %}
+            <div class="toc-entry">
+                <div class="toc-name">{{ row['Name'] }}</div>
+                <div class="toc-page-num">{{ loop.index }}</div>
+            </div>
+        {% endfor %}
+    </section>
+
+    {% for row in rows %}
+    <section class="later-pages">
+        <header>
+            <div class="header-title">{{ event.title }} / {%- trans %}Registrants Book{% endtrans -%}</div>
+            <div class="header-name">{{ row['Name'] }}</div>
+        </header>
+        <div class="participant-title">
+            <h1>{{ row['Name'] }}</h1>
+            <h1>{{ row['ID'] }}</h1>
+        </div>
+        <div class="sub-section">
+            <h2>{%- trans %}Registration Details{% endtrans -%}</h2>
+            {% set static_items_order = [
+            "Registration Date",
+            "Registration State",
+            "Price",
+            "Checked in",
+            "Check-in Date",
+            "Tags"
```

Comment:
missing indentation, and I don't particularly like having this logic inside the template nor using human-friendly column titles for this instead of internal names... it's just something that can easily break later on